### PR TITLE
Initia;

### PR DESCRIPTION
**Pull request by StasEvseev**

### CODE REVIEW

File: raid/worker.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    :return:
+    """
+    while True:
+        item = queue.get(block=True)
+        func = item['func']
+        callback = item['callback']
+        params = item['params']
+
+        res = func(**params)
+        print(os.getpid(), "got", item)
```
Comment (vesellov) [inline raid/worker.py]: shouln't we avoid `print()` in the code, @StasEvseev  ? :-)    use `if _Debug: lg.out(_DebugLevel, ...)` instead?

**Comment by vesellov** [comment]
you can also use some more descriptive name for PR...   but this is just my nagging, do not take serious :-)  👍

**Comment by StasEvseev** [comment]
Oh. Sorry. I created that only to test travis build. Sorry :)

**Comment by vesellov** [comment]
do not say sorry, Bro! it doesn't hurt :-)    Good work!

**Comment by vesellov** [comment]
it is great we have travis now! we can be lazy and do not test locally !

**Pull request closed without merging**